src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -672,5 +696,5 @@ alts_handshaker_client* alts_tsi_handshaker_get_client_for_testing(
   return handshaker->client;
 }
 
-}  // namespace internal
-}  // namespace grpc_core
+}
```

Comment:
Why do we remove these comments?

Reply:
Think it was an accident but added them back thank you!